jcache/remote/src/main/java/org/infinispan/jcache/remote/JCache.java

```diff
@@ -428,61 +428,6 @@ protected Object getCacheStatisticsMXBean() {
       return this;
    }
 
-   private JCache<K, V> checkNotNullOrNullKV(Map<? extends K, ? extends V> map, String name) {
-      checkNotNull((Object) map, name);
-      if (containsNullKey(map)) {
-         throw log.parameterMustNotContainNullKeys(name);
```

Comment:
Unfortunately the previous way never explicitly said which was null either.  It only distinguished between key or value which we still do.  I wonder is this enough still?